### PR TITLE
Ticket 36

### PR DESCRIPTION
**Pull request by cecerodrgz**
I made a new branch, the old branch was having merging issues because Main was being merged on the branch. I think this is why Kate kept seeing issues, I am hoping this fixes it!

### CODE REVIEW

**Review by jpremo — APPROVED** [review]

File: src/components/BookList/BookList.js [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
-                />
-            )}
-        </ul>
-    );
+        <section>
+            {bookList.map((item) => (
+                <BookItem item={item} />
+            ))}
+        </section>
+    )
```
Comment (jpremo) [inline src/components/BookList/BookList.js:10]: Can you revert these changes before merging?

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
Misprint at line 8. App doesn't start

File: src/components/BookList/BookList.js (hunk 1 above)
Comment (cecerodrgz) [reply src/components/BookList/BookList.js]: Just did!

**Review by Ekaterina-Bondareva — APPROVED** [review]
Great job Ceci! Nice feature! Thank you!

**Review by SimonjShurety — APPROVED** [review]
Very nice. Great work Cece.
